Cargo.toml

```diff
@@ -6,7 +6,7 @@ homepage = "https://github.com/pevers/gdcm-rs"
 repository = "https://github.com/pevers/gdcm-rs"
 readme = "README.md"
 keywords = ["dicom", "gdcm", "pixel", "encoding"]
-license = "MIT"
+license = "MIT AND BSD-2-Clause AND BSD-3-Clause AND Apache-2.0 AND OFFIS"
```

Comment:
Those licenses seem to apply to specific portions of the GDCM project. I would be more likely to interpret the `license` field in the Cargo.toml manifest as the licensing terms of the Rust bindings, rather than inheriting all licenses applicable to any portion of the project that it links to, especially because those licenses are not copy left.